transform/models/staging/_stg_models.yml

```diff
@@ -7,98 +7,98 @@ models:
     columns:
       - name: EVENT_DATE
         description: The date when the event was logged
-        data_tests:
-          - data_gaps:
-              start_date: "{{ var('ga_data_start_date') }}"
-              end_date: "current_date - 2"
+        # data_tests:
+        #   - data_gaps:
+        #       start_date: "{{ var('ga_data_start_date') }}"
+        #       end_date: "current_date - 2"
```

Comment:
Why remove this test?

Reply:
I haven't tried in my setup, but it's been [succeeding](https://california.getdbt.com/deploy/1/projects/87/runs/14851) in the nightly build.

Will take a look

Reply:
I just ran it locally in my dev environment as well from `main` and it succeeded: 

```
dbt test --select stg_ga_statewide
```

Reply:
This may be something that needs to be solved via docs: in [CI](https://california.getdbt.com/deploy/1/projects/87/runs/14887/) we run a `dbt clone` step before building to create a zero-copy clone of the large table. It's possible to run locally as well, but not well-documented here.